apps/rush-lib/src/cli/test/__snapshots__/CommandLineHelp.test.ts.snap

```diff
@@ -357,6 +357,14 @@ Optional arguments:
                         publishing workflows for older releases, providing a 
                         tag is important. When hotfix changes are made, this 
                         parameter defaults to 'hotfix'.
+  --set-access-level {public,restricted}
```

Comment:
Should the enum be `{public, restricted, preserve}`?  And then we would say that "preserve" is the default?

Reply:
Or is it obvious that you preserve by omitting this option entirely?